src/data/config/index.js

```diff
@@ -0,0 +1,14 @@
+const flagsData = require("./flags");
+const rfmosData = require("./rfmos");
+const portsData = require("./ports");
+
+module.exports = {
+  async get() {
```

Comment:
This blocks on every sub-service call, meaning all of these subservices are called in series instead of in parallel. Better to use [Promise.all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) and a single await instead.

Reply:
Good catch! I have updated the request here https://github.com/GlobalFishingWatch/api-settings/pull/2/commits/130d7ad7b1b73d00da7ec6e0a923f440682227e2